src/screens/TokenSwapReview.js

```diff
@@ -67,13 +71,26 @@ const TokenSwapReview = () => {
    */
   const exitOnError = () => {
     setModal(null);
-    dispatch(tokenSwapFetchSwapQuote(quote.direction, quote.direction === 'input' ? quote.amount_in : quote.amount_out, tokenIn.uid, tokenOut.uid));
+    dispatch(tokenSwapFetchSwapQuote(
+      quote.direction,
+      quote.direction === 'input' ? quote.amount_in : quote.amount_out,
+      tokenIn.uid,
+      tokenOut.uid
+    ));
     navigation.goBack();
   };
 
+  useEffect(() => {
+    setLoading(false);
+  }, [modal]);
```

Comment:
chore: Add an explanation for this effect, since it changes the `loading` behavior based on Modals, but there is a feedback modal that has a spinner loader below. It's not clear in the code what are the expected behaviors.

Reply:
This was to avoid leaving the loading open, but changed to a try..finally on the only place that uses setLoading(true)